### DISCUSSION TITLE
Candy Washer Dryer - ROW41066DWMCE-80

### DISCUSSION THREAD

**Discussion by jiresongit** [ofalvai/home-assistant-candy#238] [General]
Hi All,

Apologies for my naivety here, have a new candy washer dryer and have added this component and configured.

I'm seeing the following 3 sensors: 
![image](https://user-images.githubusercontent.com/78233260/210062128-704117a0-ef4c-41c9-af03-a194af616cde.png)

Which are working great, however I was hoping to be able to start cycles via Home Assistant and this Integration. Have I missed the point or is this an indication that my machine is just using the standard integration and I need to provide some extra info to have it fully supported?

Thanks
James

**Comment by ofalvai** [comment]
Hey @jiresongit,

Controlling the appliance is not yet supported, right now the integration only exposes sensors. #82 is the issue that tracks this.
